src/duplicacy_snapshotmanager.go

```diff
@@ -910,42 +927,73 @@ func (manager *SnapshotManager) CheckSnapshots(snapshotID string, revisionsToChe
                     chunkSize := chunkSizeMap[chunkID]
                     totalChunkSize += chunkSize
+                    totalChunkCount += 1
+                    if earliestSeenChunks[chunkID] == snapshot.Revision {
+                        newChunkCount += 1
+                        newChunkSize += chunkSize
+                    }
                     if chunkUniqueMap[chunkID] {
                         uniqueChunkSize += chunkSize
+                        uniqueChunkCount += 1
                     }
                 }
 
-                files := ""
+                files := " \t "
                 if snapshot.FileSize != 0 && snapshot.NumberOfFiles != 0 {
-                    files = fmt.Sprintf("%d files (%s bytes), ", snapshot.NumberOfFiles, PrettyNumber(snapshot.FileSize))
+                    files = fmt.Sprintf("%d \t%s", snapshot.NumberOfFiles, PrettyNumber(snapshot.FileSize))
                 }
-                LOG_INFO("SNAPSHOT_CHECK", "Snapshot %s at revision %d: %s%s total chunk bytes, %s unique chunk bytes",
-                         snapshot.ID, snapshot.Revision, files, PrettyNumber(totalChunkSize), PrettyNumber(uniqueChunkSize))
+                creationTime := time.Unix(snapshot.StartTime, 0).Format("2006-01-02 15:04")
+                tagWithSpace := ""
+                if len(snapshot.Tag) > 0 {
+                    tagWithSpace = snapshot.Tag + " "
+                }
+                fmt.Fprintln(tableWriter, fmt.Sprintf(
+                        "%s \t%d \t%s \t%d \t%s \t%d \t%s \t%s \t%d \t@ %s %s%s \t", 
+                        files, totalChunkCount, PrettyNumber(totalChunkSize), uniqueChunkCount, PrettyNumber(uniqueChunkSize), newChunkCount, PrettyNumber(newChunkSize), snapshotID, snapshot.Revision, creationTime, tagWithSpace, snapshot.Options))
             }
 
             var totalChunkSize int64
             var uniqueChunkSize int64
+            var totalChunkCount int64
+            var uniqueChunkCount int64
             for chunkID, _ := range snapshotChunks {
                 chunkSize := chunkSizeMap[chunkID]
                 totalChunkSize += chunkSize
+                totalChunkCount += 1
 
                 if chunkSnapshotMap[chunkID] != -1 {
                     uniqueChunkSize += chunkSize
+                    uniqueChunkCount += 1
                 }
             }
-            LOG_INFO("SNAPSHOT_CHECK", "Snapshot %s all revisions: %s total chunk bytes, %s unique chunk bytes",
-                    snapshotID, PrettyNumber(totalChunkSize), PrettyNumber(uniqueChunkSize))
+            fmt.Fprintln(tableWriter, fmt.Sprintf(
+                    " \t \t%d \t%s \t%d \t%s \t \t \t%s \tall \t \t", 
+                    totalChunkCount, PrettyNumber(totalChunkSize), uniqueChunkCount, PrettyNumber(uniqueChunkSize), snapshotID))
         }
+        tableWriter.Flush()
+        LOG_INFO("SNAPSHOT_CHECK", tableBuffer.String())
     }
 
     return true
 
 }
 
+func min(x, y int) int {
```

Comment:
Can you move this function to duplicacy_utils.go?  All utility/auxiliary functions are there.

Reply:
Sounds good.